devel/addon/ingressnginx/BUILD.bazel

```diff
@@ -1,19 +1,19 @@
 load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
 
-# Ingress image that works on Kubernetes v1.16 - v1.20
+# Ingress image that works on Kubernetes older than v1.22
 container_bundle(
-    name = "bundle_v0.48.1",
+    name = "bundle_v0.49.3",
```

Comment:
nit: would we be better to call this say, `bundle_pre_1.22` and the other just `bundle`? Just thinking that these names will change less frequently then, and it's clearer when referencing them why/what you are referencing